Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.10
+
+EXPOSE 8192
+
+ENV GOPATH /go
+WORKDIR /go/src/github.com/brave/go-update/
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
should we lock to a specific version of dep to avoid future breakage?
See here: https://github.com/brave-intl/bat-go/blob/master/Dockerfile

Reply:
Thanks, borrowed some stuff from that